edb/lib/ext/auth.edgeql

```diff
@@ -70,12 +70,12 @@ CREATE EXTENSION PACKAGE auth VERSION '1.0' {
             create annotation std::description :=
                 "The signing key used for auth extension. Must be at \
                 least 32 characters long.";
-            set default := "00000000000000000000000000000000";
```

Comment:
You never checked for this default in Python code?

Reply:
Nope! I was treating that as valid hoping that optional config values would make it moot at some point.